staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1156,6 +1156,15 @@ type ISCSIVolumeSource struct {
 	// <target portal>:<volume name> will be created for the connection.
 	// +optional
 	InitiatorName *string `json:"initiatorName,omitempty" protobuf:"bytes,12,opt,name=initiatorName"`
+	// ReplacementTimeout controls how long the iSCSI layer should wait for a
+	// timed-out path/session to reestablish itself before failing any commands on it.
+	// If the ReplacementTimeout value is 0, IO will be failed immediately.
+	// If the ReplacementTImeout value is less than 0, IO will remain queued until the session
+	// is logged back in, or until the user runs the logout command.
+	// If the ReplacementTimeout field is not provided or empty then it remains with
+	// default value i.e 120 seconds.
+	// +optional
+	ReplacementTimeout *int32 `json:"replacementTimeout,omitempty"`
```

Comment:
Please add protobuf tags.

Reply:
protobuf tags are automatically added in autogenerated files i.e in the second commit.